### PR TITLE
VictoriaMetrics Cluster first pr

### PR DESCRIPTION
**Pull request by denisgolius**
## BACKGROUND
* Add information about the background and reason for the change.

-----------------------------------------------------------------------

## Changes
* List the changes that you made

-----------------------------------------------------------------------

## Checklist
- [x] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

**Comment by jgannondo** [comment]
Hi @denisgolius -- been pretty swamped this week as you might imagine :) but we will take a look this coming week. Thanks for your patience.

**Comment by pachadotdev** [comment]
I get this error when deploying
```
error: unable to read URL "https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/victoria-metrics-cluster/yaml/vmcluster.yaml", server reported 404 Not Found, status code=404
```

**Comment by denisgolius** [comment]
> I get this error when deploying
> 
> ```
> error: unable to read URL "https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/victoria-metrics-cluster/yaml/vmcluster.yaml", server reported 404 Not Found, status code=404
> ```

Hi @pachadotdev !
It's not merged yet thats why you get 404.
Try link from my fork https://github.com/denisgolius/marketplace-kubernetes/blob/victoria-metrics-cluster-first-pr/stacks/victoria-metrics-cluster/yaml/vmcluster.yml

**Comment by denisgolius** [comment]
@jgannondo any updates?

File: stacks/victoria-metrics-cluster/deploy-local.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```
Comment (keladhruv) [inline stacks/victoria-metrics-cluster/deploy-local.sh:1]: Can you make the `deploy-local.sh` a part of the `deploy.sh` file. 
eg - [deploy.sh](https://github.com/digitalocean/marketplace-kubernetes/blob/master/stacks/kube-prometheus-stack/deploy.sh)

**Review by keladhruv — CHANGES_REQUESTED** [review]
Hi @denisgolius,

Thanks for the PR, I could get VictoriaMetrics to run locally and it looks good. I have added a comment for one change. 

Looking forward to getting VictoriaMetrics into DigitalOcean Marketplace!

Thanks!

File: stacks/victoria-metrics-cluster/deploy-local.sh (hunk 1 above)
Comment (denisgolius) [reply stacks/victoria-metrics-cluster/deploy-local.sh]: Hi @ keladhruv ! 
Done

**Review by keladhruv — APPROVED** [review]

**Pull request merged**